src/main.rs

```diff
@@ -4,16 +4,76 @@ use cairo_foundry::cli::{
 	formatter::{self, Formatter},
 };
 use clap::Parser;
+use confy;
 use log::error;
+use serde_derive::{Deserialize, Serialize};
+
+const CONFIGURATION_FILENAME: &str = "config.toml";
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+struct Config {
+	args: String,
+}
+
+/// Tries to retrieve the Args from the CLI. If the user left the CLI empty, tries to
+/// retrieve the args from the configuration file.
+fn get_args() -> cli::Args {
```

Comment:
Can you move it in a different file, in order not to cluster main.rs

Reply:
Yeah didn't know what the guideline were for `cairo-foundry`: will do just that :)

Reply:
ae3a980947f4ee64a932610637d6c54aadd24afa